assets/packs/data_table/App.js

```diff
@@ -275,6 +277,14 @@ export function App({ ctx, data }) {
     [content],
   );
 
+  const mapCellData = (cellKind, cellData) => {
+    if (cellKind === GridCellKind.Image && !Array.isArray(cellData)) {
```

Comment:
The data we pass is always a string, so I think we don't need to check for the array case.

```suggestion
    if (cellKind === GridCellKind.Image) {
```

Reply:
When writing that i thought that maybe we could use the fact that our frontend data grid library supports displaying multiple images in a cell, and provide an API for this later on in the elixir side. But yeah that would require re-thinking that system where currently we only pass strings to the frontend.

Reply:
On second thought, yeah, it might be unnecessary for now.